README.md

```diff
@@ -45,6 +45,24 @@ app.start(port=8080)
 
 ```
 
+## Python Version Support
+
+Robyn is compatible with the following Python versions:
+
+> Python >= 3.6
```

Comment:
Thanks for the PR @Oluwaseun241 😄 

But Robyn only supports Python version >= 3.7.

Also, would it be possible to add some sort of version shield? Something like this - https://shields.io/

Reply:
I would look into that and update the version support